app/controllers/catalog_controller.rb

```diff
@@ -1890,6 +1901,8 @@ def replace_right_cell(action = nil, replace_trees = [])
     end
     presenter[:right_cell_text] = right_cell_text
 
+    @generic_subtypes = generic_item_subtypes
```

Comment:
@h-kataria I need to set it when I select a Service to edit. But I may look for a nicer place for it.